tests/foreman/ui/test_containerimagetag.py

```diff
@@ -25,18 +24,18 @@
 
 
 @pytest.fixture(scope="module")
-def module_org():
-    return entities.Organization().create()
+def module_org(module_target_sat):
```

Comment:
Same here

Reply:
this was resolved without comments, are we good here?

Reply:
please refer this comment for this https://github.com/SatelliteQE/robottelo/pull/13809#discussion_r1464795332